LOL That is a weird failure if sporadic:

```pytb
  _______________________________ test_screenshot ________________________________
  
  make_napari_viewer = <function make_napari_viewer.<locals>.actual_factory at 0x1348fd080>
  qapp = <PyQt6.QtWidgets.QApplication object at 0x32ee82c10>
  qtbot = <napari.conftest.QtBotWithOnCloseRenaming object at 0x32bd53b70>
  
      @skip_on_win_ci
      def test_screenshot(make_napari_viewer, qapp, qtbot):
          "Test taking a screenshot"
          viewer = make_napari_viewer(show=True)
          rng = np.random.default_rng(0)
          # Add image
          data = rng.random((10, 15))
          viewer.add_image(data)
      
          # Add labels
          data = rng.integers(20, size=(10, 15))
          viewer.add_labels(data)
      
          # Add points
          data = 20 * rng.random((10, 2))
          viewer.add_points(data)
      
          # Add vectors
          data = 20 * rng.random((10, 2, 2))
          viewer.add_vectors(data)
      
          # Add shapes
          data = 20 * rng.random((10, 4, 2))
          viewer.add_shapes(data)
          # wait for window be fully rendered
          qtbot.wait_exposed(qt_viewer)
      
          # without these two lines, the shape of screenshot1 and screenshot2 differ by
          # 2 pixels. It looks like some event requires inactivity in event loop to
          # trigger resize to final shape.
          qtbot.wait(5)
          qapp.processEvents()
      
          screenshot2 = viewer.window.screenshot(flash=False, canvas_only=True)
      
          qapp.processEvents()
      
          # Take screenshot
          with pytest.warns(FutureWarning, match='qt_viewer'):
              screenshot1 = viewer.window.qt_viewer.screenshot(flash=False)
      
  >       npt.assert_array_equal(screenshot1, screenshot2)
  E       AssertionError: 
  E       Arrays are not equal
  E       
  E       Mismatched elements: 2 / 753080 (0.000266%)
  E       Max absolute difference among violations: 1
  E       Max relative difference among violations: 0.01030928
  E        ACTUAL: array([[[  0,   0,   0, 255],
  E               [  0,   0,   0, 255],
  E               [  0,   0,   0, 255],...
  E        DESIRED: array([[[  0,   0,   0, 255],
  E               [  0,   0,   0, 255],
  E               [  0,   0,   0, 255],...
  
  src/napari/_qt/_tests/test_viewer_qt_integration.py:232: AssertionError
```

Note **`Mismatched elements: 2 / 753080`** — that is half a pixel's worth of mismatched elements. And they are off by 1. What.

I've rerun just in case. Thanks @psobolewskiPhD and sorry for muddying the waters about dropping this — I understand the issue and fix better now. 😅

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -128,7 +128,7 @@ optional-numba = [
 ]
 optional = [
     "napari[optional-base,optional-numba,bermuda]",
-    "triangle; python_version <= '3.13'",  # there is no wheel for triangle on python 3.14 at the time and build from source causes problems. 
+    "triangle; python_version <= '3.13'",  # there is no wheel for triangle on python 3.14 at the time and build from source causes problems.
 ]
 bermuda = [
     "bermuda>=0.1.5",

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -128,7 +128,7 @@ optional-numba = [
 ]
 optional = [
     "napari[optional-base,optional-numba,bermuda]",
-    "triangle; python_version <= '3.13'",
+    "triangle; python_version <= '3.13'",  # there is no wheel for triangle on python 3.14 at the time and build from source causes problems. 
 ]
 bermuda = [
     "bermuda>=0.1.5",

